build.gradle

```diff
@@ -18,9 +18,9 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.9.1'
 
     // logging
-    implementation 'org.slf4j:slf4j-api:2.0.1'
```

Comment:
I saw your PR for version changes. I think we can revert the version changes here and keep the dependency issues in that other PR.

Reply:
I agree, removed this commit as we have another PR for it.